From 824d94238fdf86989778d634d5f5eed7d5cfb200 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Fri, 22 Jul 2022 10:59:27 +0800
Subject: [PATCH] drivers: ram: k3-ddrss: drop the whole file cps_bm

Drop the whole file drivers/ram/k3-ddrss/cps_bm.c, since there is
license limitation for this file. And the file is no use for current u-boot
building.

(LOCAL REV: NOT UPSTREAM) -- Not applicable

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/ram/k3-ddrss/cps_bm.c | 126 ----------------------------------
 1 file changed, 126 deletions(-)
 delete mode 100644 drivers/ram/k3-ddrss/cps_bm.c

diff --git a/drivers/ram/k3-ddrss/cps_bm.c b/drivers/ram/k3-ddrss/cps_bm.c
deleted file mode 100644
index f26dcc9efb9..00000000000
--- a/drivers/ram/k3-ddrss/cps_bm.c
+++ /dev/null
@@ -1,126 +0,0 @@
-/******************************************************************************
- * Copyright (C) 2014-2018 Cadence Design Systems, Inc.
- * All rights reserved worldwide
- *
- * The material contained herein is the proprietary and confidential
- * information of Cadence or its licensors, and is supplied subject to, and may
- * be used only by Cadence's customer in accordance with a previously executed
- * license and maintenance agreement between Cadence and that customer.
- *
- ******************************************************************************
- * cps.c
- *
- * Sample implementation of Cadence Platform Services for a bare-metal
- * system
- ******************************************************************************
- */
-
-
-#include "cps.h"
-#include <string.h>
-#define _weak __attribute__((weak))
-
-
-/* see cps.h */
-_weak uint32_t cps_readreg32(volatile uint32_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak void cps_writereg32(volatile uint32_t* address, uint32_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak uint8_t cps_uncachedread8(volatile uint8_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak uint16_t cps_uncachedread16(volatile uint16_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak uint32_t cps_uncachedread32(volatile uint32_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak uint64_t cps_uncachedread64(volatile uint64_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak uint64_t cps_readreg64(volatile uint64_t* address) {
-    return *address;
-}
-
-/* see cps.h */
-_weak void cps_uncachedwrite8(volatile uint8_t* address, uint8_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak void cps_uncachedwrite16(volatile uint16_t* address, uint16_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak void cps_uncachedwrite32(volatile uint32_t* address, uint32_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak void cps_uncachedwrite64(volatile uint64_t* address, uint64_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak void cps_writereg64(volatile uint64_t* address, uint64_t value) {
-    *address = value;
-}
-
-/* see cps.h */
-_weak void cps_writephysaddress32(volatile uint32_t* location, uint32_t addrValue) {
-    *location = addrValue;
-}
-
-/* see cps.h */
-_weak void cps_buffercopy(volatile uint8_t *dst, volatile const uint8_t *src, uint32_t size) {
-    memcpy((void*)dst, (void*)src, size);
-}
-
-/* see cps.h */
-_weak void cps_cacheinvalidate(void* address, size_t size, uintptr_t devInfo) {
-    (void) address;
-    (void) size;
-    (void) devInfo;
-}
-
-/* see cps.h */
-_weak void cps_cacheflush(void* address, size_t size, uintptr_t devInfo) {
-    (void) address;
-    (void) size;
-    (void) devInfo;
-}
-
-/* see cps.h */
-_weak void cps_delayns(uint32_t ns) {
-    (void) ns;
-}
-
-/* see cps.h */
-_weak void cps_memorybarrier(void) {
-
-}
-
-/* see cps.h */
-_weak void cps_memorybarrierwrite(void) {
-
-}
-
-/* see cps.h */
-_weak void cps_memorybarrierread(void) {
-
-}
-- 
2.36.1

